docs/manual/common/concepts/DevelopmentEnvironmentOverview.md

```diff
@@ -0,0 +1,26 @@
+# Development environment overview
+
+Setting up a Lagom development environment takes only minutes using an sbt template or Maven archetype as described in [[Getting Started with Lagom|IntroGetStarted]]. Without any coding, scripting or configuration on your part, the templates set up the following:
+
+* A hierarchical build structure with two example services. You can follow their pattern to begin creating your own services, right in the same project.
+
+* The infrastructure necessary to run and test your services locally:
+    
+    * A Kafka server for handling messages. 
+
+    * A Cassandra server for handling persistence.  
+
+    * A Service Registry and Service Gateway to support location transparency. 
+
+* The scripts necessary to build and run the services and infrastructure.
```

Comment:
I was trying to make the point that the user doesn't really have to do anything. Is there a better way to say this?